storage/Btrfs.cc

```diff
@@ -848,20 +848,81 @@ Btrfs::getCommitActions(list<commitAction>& l) const
     }
 
 
+    int
+    Btrfs::extraMount()
+    {
+	y2mil("extraMount");
+
+	if (getMount() == "/")
+	{
+	    string def_subvol = getStorage()->getDefaultSubvolName();
+
+	    list<string> ign_opt(ignore_opt, ignore_opt + lengthof(ignore_opt));
+	    list<string> ign_beg(ignore_beg, ignore_beg + lengthof(ignore_beg));
+
+	    if (getStorage()->instsys())
+		ign_opt.push_back("ro");
+
+	    ign_beg.push_back("subvol=");
+
+	    list<string> opts = splitString(fstab_opt, ",");
+
+	    y2mil("opts before:" << opts);
+	    for (const string& tmp : ign_opt)
+		opts.remove(tmp);
+	    for (const string& tmp : ign_beg)
+		opts.remove_if(string_starts_with(tmp));
+	    y2mil("opts after:" << opts);
+
+	    for (const Subvolume& subvolume : subvolumes)
+	    {
+		// TODO workaround
```

Comment:
for new workarounds it would be nice to point to bug report or some date. So it is easier to verify in future if workaround is still needed and can remove it if it is no longer needed.